collections/index.js

```diff
@@ -20,46 +20,130 @@ import employees from './data.json';
   go for it - that's the whole notion of functional programming
 
   The format for this test will be slightly different - the data that I will
-  be using for the test suite lives in ./data.json. I have written you an 
+  be using for the test suite lives in ./data.json. I have written you an
   import statement and you should use these data sets for the test.
 */
+export const howManyEmployees = () => {
+  return employees.length;
+}
 
 /*
   I expect this function to get me an employee by their ID
 */
 export const getEmployeeById = (id) => {
-
+  // created an empty object to insert data into
+  var employeeNumber = {};
+  // using a for loop to index through each of the employees on record
+  for (let employeeNumberIndex = 0; employeeNumberIndex < employees.length; employeeNumberIndex++) {
```

Comment:
While being verbose is generally a good thing, in loops it's common practise to just define the loop index as `i`

```js
for (let i = 0; i < arr.length; i++) {
  ...
}
```

Reply:
done - I like the idea of being verbose to start off with because it helps with my understanding of whats going on